Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/PatientCorrelationOrchImpl.java

```diff
@@ -253,4 +254,21 @@ private static Date getExpirationDate(String expirationUnits, int expiration) th
         }
         return calendar.getTime();
     }
+
+    @Override
+    public  void addPatientCorrelationPLQ(PatientLocationQueryResponseType plqRecords,
+        AssertionType assertion) {
+
+        if (plqRecords == null) {
```

Comment:
If we are checking for Null here and its undesired, why still call the addPatientCorrelationPLQRecords method?